site/src/examples/border-layout/index.module.css

```diff
@@ -0,0 +1,12 @@
+.borderItem {
+  --borderLayout-border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-secondary-borderColor);
```

Comment:
Does this need to be saved in a variable?

Reply:
Nope 😄